diff -up librtas-1.3.4/rules.mk.libdir librtas-1.3.4/rules.mk
--- librtas-1.3.4/rules.mk.libdir	2009-09-21 16:41:57.000000000 +0200
+++ librtas-1.3.4/rules.mk	2009-09-21 16:42:01.000000000 +0200
@@ -135,7 +135,7 @@ define install_sbin
 endef
 
 define install_lib
-	$(call install_files,$1,755,$2/$(LIB_DIR)$(findstring 64,$(shell file $(firstword $1))))
+	$(call install_files,$1,755,$2/$(LIB_DIR))
 endef
 
 define install_inc
